content/en/templates/taxonomy-templates.md

```diff
@@ -238,7 +253,7 @@ To list such taxonomies, use the following:
 ```
 {{ if .Params.directors }}
   <strong>Director{{ if gt (len .Params.directors) 1 }}s{{ end }}:</strong>
-  {{ range $index, $director := .Params.directors }}{{ if gt $index 0 }}, {{ end }}<a href="{{ "directors/" | relURL }}{{ . | urlize }}">{{ . }}</a>{{ end }}
+  {{ range $index, $director := .Params.directors }}{{ if gt $index 0 }}, {{ end }}{{ with $.Site.GetPage "taxonomy" (printf "directors/%s" $director) }}<a href="{{ .Permalink }}">{{ $director }}</a>{{ end }}{{ end }}
```

Comment:
While I see that you are trying to keep the original form, please break up the statements on separate lines. Do this consistently everywhere:

```html
{{ range .. }}
    {{ with .. }}
        {{ .. }}
    {{ end }}
{{ end }}
``` 

Use `{{`/`}}` or `{{-`/`-}}` based on how you want to manage the spaces in the final output.